packages/migrate/src/commands/DbPull.ts

```diff
@@ -163,41 +163,47 @@ Set composite types introspection depth to 2 levels
      *
      * If neither these variables were set, we'd have already thrown a `NoSchemaFoundError`.
      */
-    const schema = await match({ url, schemaPath })
+    const { config, schema } = await match({ url, schemaPath })
```

Comment:
Centralized `getConfig` usage here, while preserving the pre-existing behavior. This came handy while inspecting `input.url` protocol validity (see TODOs below), and reduces redundant calls on the same arguments.